guardian/french-thrift
# Sync with upstream v0.19.0

## What does this change?

- Syncs with the upstream repository `apache/thrift` `0.19.0` branch
- Removes reserved keywords, similar to 3bae6d21037691e1bf4fec84f16af0bd6bfa925b. See https://github.com/guardian/french-thrift/pull/12#issuecomment-1804240046.

## How to test

- [x] Before merging, create a prerelease from this branch ([here](https://github.com/guardian/french-thrift/releases/tag/v0.19.0-prerelease))
- [ ] Create a branch in [Bridget](https://github.com/guardian/bridget) to check this prerelease works
  - Set the `validate-thrift` action's Dockerfile to use the prerelease: https://github.com/guardian/bridget/blob/main/.github/actions/validate-thrift/Dockerfile#L3
  - Disable the [paths filter ](https://github.com/guardian/bridget/blob/main/.github/workflows/validate-thrift.yml#L8) for the `validate-thrift` workflow so it runs for the test Pull Request

## Further details

I followed the directions outlined in the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-command-line) for syncing a fork with the upstream repository from the command line.
